scrapy/templates/project/module/middlewares.py.tmpl

```diff
@@ -5,6 +5,9 @@
 
 from scrapy import signals
 
+# useful for handling different item types with a single interface
+from itemadapter import ItemAdapter
```

Comment:
For spider middlewares, `is_adapter` can be handy as well. Also, we need to update the comments below to replace `dict or Item objects` with `` :ref:`item objects <topics-items>` ``.